u2u-testnet/logo.svg

```diff
@@ -0,0 +1,17 @@
+<svg width="1085" height="120" viewBox="0 0 1085 120" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Comment:
please fix to include just the logo + run through an svg minifier  https://jakearchibald.github.io/svgomg/